Tools/Scripts/find-feature-flags.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
```

Comment:
Can we make this Python 3? And we usually drop the `.py` extension on scripts the user invokes directly.

Reply:
Yea sorry Windows doesn't do shebangs so its a force of habit to give actual extensions.